charts/refinery/values.yaml

```
@@ -176,7 +176,7 @@ config:
     # This value should be set in according to the resources.limits.memory
     # By default that setting is 2GB, and this is set to 85% of that limit
     # 2 * 1024 * 1024 * 1024 * 0.80 = 1,717,986,918
-    MaxAlloc: 1717986918
+    # MaxAlloc: 42
```

I don't think we need to update this value

```suggestion
    # MaxAlloc: 1717986918
```

---

Agree. This was a little joke for Hitchhiker's fans that I intended to pull after exiting draft.